skatepark/patterns/mixed-media-in-container.php

```diff
@@ -27,7 +27,7 @@
 <div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
 <!-- /wp:spacer -->
 
-<!-- wp:image {"sizeSlug":"large","style":{"color":{"duotone":["#000","#BFF5A5"]}}} -->
+<!-- wp:image {"sizeSlug":"large"} -->
```

Comment:
I checked the `theme.json` for skatepark and see [`core/image` is using the default duotone filter](https://github.com/Automattic/themes/blob/trunk/skatepark/theme.json#L312) which is [`["#000","#B9FB9C"]`](https://github.com/Automattic/themes/blob/trunk/skatepark/theme.json#L37). This image uses a slightly different (more muted) green of `#BFF5A5`. I'm not sure if that was originally intentional or a mistake. I don't think anyone would notice the change, but I wanted to point it out here to make sure.

Reply:
I see a precedent [for changing from `#BFF5A5` to `#B9FB9C`](https://github.com/Automattic/themes/commit/f1a3133076e478decfcb97e2efb9f47198d015c4#diff-c6ba9c8ac9c93710eaec5f8bac0f44d1d874aaa5c1d289dd72b38960726909eaL32), and honestly can't imagine someone would visually notice the distinction between the two, so I don't think this should be a blocker.